browser/ntp_background/new_tab_takeover_infobar_delegate.cc

```diff
@@ -24,7 +24,7 @@
 namespace ntp_background_images {
 
 // static
-void NewTabTakeoverInfoBarDelegate::MaybeCreate(
+void NewTabTakeoverInfoBarDelegate::MaybeShowAndRecordInfobarShown(
```

Comment:
nit: `MaybeShowInfobarAndCount`

Reply:
how about `MaybeDisplayAndIncrementCounter`?

Reply:
Fixed

Reply:
Sorry to jump in here too, but the new name confused me - we actually decrement a counter in `RecordNewTabTakeoverInfobarWasShown`